tasks/java-RedHat.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: add Azul Systems Java yum repository
+  yum_repository:
+    name: zulu
+    description: zulu-$releasever - Azul Systems Inc., Zulu packages for $basearch
```

Comment:
Are the `$releasever` parameters supposed to be resolved? AFAIK they're not valid Ansible/Jinja2 placeholders.

Reply:
This is something that Yum resolves. For example, this is an excerpt from one of CentOS's default yum repo files:

```
$ cat /etc/yum.repos.d/CentOS-Base.repo                                                                                                                                                                                     [11/723]
# CentOS-Base.repo                                 
#
# The mirror system uses the connecting IP address of the client and the
# update status of each mirror to pick mirrors that are updated to and
# geographically close to the client.  You should use this for CentOS updates
# unless you are manually picking other mirrors.                                                        
#                                                                         
# If the mirrorlist= does not work for you, as a fall back you can try the
# remarked out baseurl= line instead.
#                                                  
#
                      
[base]
name=CentOS-$releasever - Base
mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
#baseurl=http://mirror.centos.org/centos/$releasever/os/$basearch/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

#released updates
[updates]
name=CentOS-$releasever - Updates
mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
#baseurl=http://mirror.centos.org/centos/$releasever/updates/$basearch/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```